tests/test_views.py

```diff
@@ -34,7 +34,7 @@ def test_resource_card_resource_no_schema(self):
         assert '' == render_resource_card(resource=ResourceFactory(schema=None))
 
     def test_resource_card_resource_with_schema(self):
-        resource = ResourceFactory(schema='etalab/irve')
+        resource = ResourceFactory(schema={"name":"etalab/irve"})
```

Comment:
```suggestion
        resource = ResourceFactory(schema={'name': 'etalab/irve'})
```

sorry, now that it's not a string... 😬

Reply:
BTW I think using flake8 as a linter could help you with some of my usual comments (but not double quotes vs singles, this is strictly a convention).